src/contexts/CampaignsContext/CampaignsDataContext.tsx

```diff
@@ -37,6 +37,8 @@ type CamapignBackendDataRes = Campaign & {
   clicks?: number
 }
 
+type StatusToChange = 'resume' | 'close' | 'pause'
```

Comment:
Why this `StatusToChange`, when you have enum `CampaignStatus`
e.g.
`changeCampaignStatus(CampaignStatus.close, id)`

Reply:
doing this will remove the need of `getCampaignStatus`